recipes/lens-vpr/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "lens-vpr" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lens-vpr-{{ version }}.tar.gz
+  sha256: 5e706367882ff2701e2de708a46a7774f457f1d3d793672712e0b7fea510dbf5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6, !=3.12.* 
```

Do you know this will work with 3.13? If it doesn't work with 3.12 there is a big chance that it won't work with any >=3.12.

```suggestion
    - python >=3.6,<3.12
```